docs/user-guide/DREAM-SANS-methodology.ipynb

```diff
@@ -0,0 +1,56 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "788a4fce-63dc-4c23-8d2a-e9a05db0fdf7",
+   "metadata": {},
+   "source": [
+    "# DREAM-SANS methodology\n",
+    "\n",
+    "The general data reduction scheme presented in \"SANS Polarization Analysis Methodology\" will be adapted for the SANS option at the DREAM instrument at the ESS. Following considerations have to be changed with respect to the general methodology:\n",
+    "\n",
+    "### Polarizer:<br>\n",
+    "The DREAM-polarizer is a movable in-situ SEOP <sup>3</sup>He-cell of which the neutron polarization $P^{^3\\textup{He, p}}$ has to be pre-characterized in commissioning measurements and inserted into this workflow. Its wavelength-independent opacity $O^{\\textup{0, p}}$, empty-glass transmission $T^{\\textup{g, p}}$, pre-factor of nuclear polarization $C^{\\textup{p}}$, and its time-decay constant $T^{\\textup{1, p}}$ can be measured once in every commissioning time via Workflow 2 of \"SANS Polarization Analysis Methodology\". The resulting $P^{^3\\textup{He, p}}$ can be inserted into the workflow as constant parameter. To account for sudden fluctuations of $P^{^3\\textup{He, p}}$, which can occur due to changes in pressure or the like, the beam intensity after the polarizer has to be continuously recorded as function of time. Sudden fluctuations in intensity above a pre-defined difference in time must pop out an error message \"Critical intensity fluctuations detected in polarizer-monitor. Re-measure $P^{^3\\textup{He, p}}$!\".<br>\n",
```

Comment:
Can you explain what `SEOP` stands for?

Reply:
> `$P^{^3\\textup{He, p}}$`

I am not sure what the `p` superscript stands for? I'm not sure I've seen it before.

Reply:
> Sudden fluctuations in intensity above a pre-defined difference in time must pop out an error message

Is it the frequency of fluctuations that is important, or is it the amplitude of them, or both?
Also, "pop out" -> "raise".

Reply:
> Can you explain what `SEOP` stands for?
- I am adding explanations for SEOP and MEOP cells
> I am not sure what the `p` superscript stands for? I'm not sure I've seen it before.
- It stands for polarizer ("p") and analyzer ("a"). It was defined in sans-polarization-analysis-methodology.ipynb for the transmissions of the polarizer and analyzer (Tp, and Ta, respectively), but yes I will define them here again!
> Is it the frequency of fluctuations that is important, or is it the amplitude of them, or both? Also, "pop out" -> "raise".
- the amplitude